src/main/java/seedu/address/storage/ReminderPersons.java

```diff
@@ -33,33 +33,54 @@ public static ReminderPersons getInstance() {
      * @param person The person to be reminded of.
      */
     public Reminder add(Person person, Reminder reminder) {
-        return reminderPersons.reminders.put(person, reminder);
+        return reminders.put(person, reminder);
     }
 
     /**
      * Removes a person from the list of persons to be reminded of
      * @param person The person to remove from reminders.
      */
     public Reminder remove(Person person) {
-        return reminderPersons.reminders.remove(person);
+        return reminders.remove(person);
```

Comment:
Not really sure what `reminders` here represent, but I am assuming it to be a list that contains the clients to remind. If so, I would suggest to change the name as it does not logically make sense for `reminders` to remove a `Person`. Similarly for the methods below (if this applies).

Reply:
got it, adjusted the name accordingly 👍